index.js

```diff
@@ -113,7 +113,11 @@ Mailgen.prototype.parseParams = function (params) {
 
     // Use `greeting` and `name` for title if not set
     if (!body.title) {
-        body.title = body.greeting + ' ' + body.name + ',';
+        body.title = body.greeting;
```

Comment:
@lirantal nope, there is already a check for that here:
https://github.com/eladnava/mailgen/blob/461ab1c84ad99488161deb2d71457f2b547e4f56/index.js#L111